portfolio/src/main/java/com/google/sps/data/CommentHistory.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing the subtraction game, where players take turns subtracting from 21 to reach 0.
+ *
+ * <p>Note: The private variables in this class are converted into JSON.
+ */
+public class CommentHistory {
```

Comment:
Consider renaming this class to just "Comment". "CommentHistory" implies that the class contains a list of comments, but each instance of this class really only represents just one comment.